packages/default/scss/popover/_layout.scss

```diff
@@ -0,0 +1,82 @@
+@include exports("popover/layout") {
+
+    // Popover
+    .k-popover {
+        @include border-radius( $popover-border-radius );
+        border-width: $popover-border-width;
+        border-style: $popover-border-style;
+        box-sizing: border-box;
+        outline: 0;
+        font-size: $popover-font-size;
+        font-family: $popover-font-family;
+        line-height: $popover-line-height;
+        display: flex;
+        flex-flow: column nowrap;
+        position: absolute;
+        z-index: 12000;
+        -webkit-touch-callout: none;
+        -webkit-tap-highlight-color: $rgba-transparent;
+    }
+
+    // Header
+    .k-popover-header {
+        @include border-top-radius( $popover-border-radius );
+        padding: $popover-header-padding-y $popover-header-padding-x;
+        border-width: 0 0 $popover-header-border-width;
+        border-style: $popover-header-border-style;
+    }
+
+    // Body
+    .k-popover-body {
+        padding: $popover-body-padding-y $popover-body-padding-x;
+    }
+
+    // Actions
+    .k-popover-actions {
+        border-width: $popover-actions-border-width 0 0;
+    }
+    .k-popover-actions > .k-button {
+        padding-left: $card-actions-button-padding-left;
+        padding-right: $card-actions-button-padding-right;
+        max-width: 100%;
+        white-space: normal;
+    }
+
+}
+
+@include exports( "popover/layout/callout" ) {
+
+    .k-popover-callout {
+        margin: 0;
+        width: $popover-callout-width;
+        height: $popover-callout-height;
+        border-width: $popover-callout-border-width $popover-callout-border-width 0 0;
+        border-style: $popover-callout-border-style;
+        position: absolute;
+    }
+
+    .k-popover-callout.k-callout-n {
```

Comment:
It might be better (and more consistent) if we nest the `k-callout-n/e/s/w` classes in `k-popover-callout`.
They are already nested in the _theme.scss file.